### ISSUE TITLE
Extension Blocked By Chrome

### ISSUE THREAD

**Issue by VeryNaughtyCat** [rushilsrivastava/OpenNews#3]
Hi

The extension is blocked by Chrome as it is no longer downloadable from the Chrome store.  Chrome will not let me manually enable it, even in Developer mode.

Any advice on what to do?

Many thanks.

VBC

**Comment by rushilsrivastava** [comment]
Hey,

To install the extension as of now, ensure you are in Developer Mode, then: Download the latest extension from the `Releases` section, ensure it is in the default Downloads Directory on your computer, and finally drag-and-drop it to your [extensions page](chrome://extensions).

Let me know if this works for you. I will clarify the instructions on the README to make this clearer.

**Comment by kungpao42** [comment]
I am facing the same issue in Chrome 71.  I am not able to enable the extension after following the process listed in README

**Comment by VeryNaughtyCat** [comment]
Yes, FT has stopped working today. Was fine yesterday

Thanks

**Comment by rushilsrivastava** [comment]
@kungpao42 Can you please confirm that you are in developer mode and that the `.crx` file downloaded is in the default download directory? I am on Chrome 71 as well, and I tested on both MacOS and Windows and it works for me.

**Comment by rushilsrivastava** [comment]
@VeryNaughtyCat FT does look like it's broken, can you create a new issue (so I can reference commits) there?

**Comment by kungpao42** [comment]
I fixed my issue by moving back to the normal release channel of Chrome, I was on beta.  Additionally, I am running OSX...so I deleted my preference files, etc:

rm -rf ~Library/Google

That caused the extension to load and was no longer blocked in the UI.

**Comment by rushilsrivastava** [comment]
@kungpao42 Thanks for sharing. @VeryNaughtyCat are you on the beta channel by any chance?

**Comment by bdotq** [comment]
Hi, still experiencing this issue with Chrome Beta 84.
This resolved my issue: https://www.ghacks.net/2014/05/28/unsupported-extensions-disabled-re-enable-blocked-extensions-chrome/

> extract the extension file (.crx) on your system for this to work. The easiest way to handle this is to download the extension again from the developer website. Right-click the extension again and extract it to a folder on your system... (eg with 7zip)
>Now that you have extracted the extension, click on load unpacked extension and select the folder that you have extracted the extension files to.
>The extension will be installed and won't be removed on the next start of the browser. Chrome will display a "disable developer mode extensions" notification though in the browser, but you can cancel that to skip it. Note that it will display the message on every start of the browser.
